sdk/src/main/java/com/deploygate/sdk/DeployGate.java

```diff
@@ -1390,4 +1402,192 @@ public static String getDistributionUserName() {
 
         return sInstance.mDistributionUserName;
     }
+
```

Comment:
I prefer providing `getBuildEnvironment(): SomeExtras` and `getRuntimeExtras(): SomeExtras` much more than providing many methods from DeployGate class.

`SomeExtras` above is kinda thread-safe KVS and has named interfaces like `putBoolean`, `putInt` and `putString`.

Reply:
I understand. 
If we are provide with `Some Extra` class, we can also provide builder pattern for create values. 👍